app/models/katello/glue/pulp/repo.rb

```diff
@@ -744,6 +744,13 @@ def ostree?
         self.content_type == Repository::OSTREE_TYPE
       end
 
+      def capsule_download_policy
+        if self.is_a?(::Katello::Repository)
+          repo = self.library_instance || self
+          repo.download_policy
+        end
+      end
```

Comment:
This function should just live in the repository model itself and not the glue layer since it is a database attribute. Your new test made me realize that (since it was in the model unit test and not glue unit test).